modules/lustre/main.tf

```diff
@@ -87,6 +88,7 @@ resource "aws_cloudwatch_log_group" "this" {
   retention_in_days = var.cloudwatch_log_group_retention_in_days
   kms_key_id        = var.cloudwatch_log_group_kms_key_id
   log_group_class   = var.cloudwatch_log_group_class
+  skip_destroy      = var.cloudwatch_log_group_skip_destroy
```

Comment:
lets remove this - it doesn't make sense to skip log groups, it will only cause conflicts